.github/workflows/main.yml

```
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+       node-version: '14.x'
+
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+
```

Let's add Python and yarn caching
```suggestion

    - uses: actions/cache@v2
      with:
        path: ~/.cache/pip
        key: ${{ runner.os }}-pip-${{ hashFiles('**/package.json') }}
        restore-keys: |
          ${{ runner.os }}-pip-
          
    - name: Get yarn (jlpm) cache directory path
      id: yarn-cache-dir-path
      run: echo "::set-output name=dir::$(jlpm cache dir)"

    - uses: actions/cache@v2
      id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
      with:
        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
        restore-keys: |
          ${{ runner.os }}-yarn-

```

---

oh wait, I need to do the yarn caching after the jlab install so it has jlpm....

---

I'll just submit a commit, which is easier.